velero-csi-plugin/csi_snapshotter.go

```diff
@@ -34,6 +36,11 @@ import (
 	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
 )
 
+const (
+	veleroBackupNameAnnotation   = "velero.io/backup-name"
```

Comment:
this constant is defined in the velero `pkg/apis/velero/v1` package, can probably reference it from there